src/web/mjs/connectors/Dmzj.mjs

```diff
@@ -0,0 +1,36 @@
+import SinMH from './templates/SinMH.mjs';
+
+export default class Dmzj extends SinMH {
+
+    constructor() {
+        super();
+        super.id = 'dmzj';
+        super.label = '动漫之家(DMZJ)';
+        this.tags = [ 'manga', 'chinese', 'webtoon' ];
+        this.url = 'https://www.dmzj.com';
+        this.requestOptions.headers.set('x-referer', this.url);
+
+        this.queryChapters = 'div.tab-content-selected ul li a';
+        this.queryManga = 'div.comic_deCon > h1';
+        this.path = '/category/0-0-0-0-0-0-%PAGE%.html';
+        this.queryMangasPageCount = 'a.pg_last';
+        this.queryMangas = 'h3 > a';
+
+        this.queryPagesScript =`
+            new Promise(resolve => {
+                resolve(picArry.map(el => new URL(el,img_prefix).href));
+            });
+        `;
+    }
+
+    async _getChapters(manga) {
```

Comment:
Is this method required, i would expect the one from the base class to work correctly

Reply:
i figured that as well, but i wasn't able to get it to work

Reply:
nvm

Reply:
When injecting the script from the base class into the browser it works fine

![image](https://user-images.githubusercontent.com/27136773/97800889-1a023d00-1c39-11eb-9856-d55ea8ac173c.png)